### PR TITLE
feat: added rating component

### PR DESCRIPTION
**Pull request by eakansh125**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -635,3 +678,106 @@ main{
...
+}
+
+/* List */
+
+.list-stacked{
+    background-color: var(--secondary-grey);
+    display: flex;
+    align-items: center;
+    justify-content: space-evenly;
+    border: 1px solid var(--light-grey);
```
Comment (parvez-rashid) [inline style.css:777]: Variables `secondary-grey` and  `light-grey` might confuse people due to similar names. You can use names like `grey-100`, `grey-400` to denote darkness.

File: style.css [hunk 2]
```diff
@@ -635,3 +678,106 @@ main{
...
+    width: 10rem;
+    cursor: pointer;
+    padding: 0.5rem;
```
Comment (parvez-rashid) [inline style.css:780]: Use px for values less than 1rem.

**Review by parvez-rashid — APPROVED** [review]
UI Review: I like the clean UI, looks great. I have some comments below on code, other than that the code looks well-structured. Can be merged.

File: style.css (hunk 1 above)
Comment (eakansh125) [reply style.css:777]: Okay, good point, will keep that in mind and try to change accordingly later

File: style.css (hunk 2 above)
Comment (eakansh125) [reply style.css:780]: Will keep that in mind, thank you!

File: rating.html [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+            <div class="nav-header-div-lt">
+                <a class="text-dec-none font-bold margin-lr font-primary-color" href="index.html">Home</a>
+                <a class="text-dec-none font-bold margin-lr font-primary-color" href="docs.html">Docs</a>
+                <a class="text-dec-none font-bold margin-lr font-primary-color" href="#">About us</a>
+                <a class="text-dec-none font-bold margin-lr font-primary-color" href="#">Blogs</a>
+            </div>
+            <div class="nav-header-div-rt">
+                <a href="https://github.com/eakansh125/Fleet-UI/tree/dev"><img class="margin-lr"
+                        src="https://img.icons8.com/ios-filled/20/000000/github.png" /></a>
+                <a href="#"><img class="margin-lr"
```
Comment (JuvitaSaini09) [inline rating.html:24]: use alt attribute

**Review by JuvitaSaini09 — APPROVED** [review]
code looks clean and good to merge

File: rating.html (hunk 3 above)
Comment (eakansh125) [reply rating.html:24]: sure

**Review by Sahil25061999 — APPROVED** [review]
Looks clean to me, no major changes required

File: rating.html (hunk 3 above)
Comment (eakansh125) [reply rating.html:24]: done, thank you!

**Pull request merged**